modules/core/shared/src/main/scala/lucuma/core/math/Wavelength.scala

```diff
@@ -79,7 +81,7 @@ object Wavelength {
     * Construct a wavelength from a positive int
     * @group constructor
     */
-  def apply(picometers: PosInt): Wavelength =
+  def apply(picometers: PosInt)(using DummyImplicit): Wavelength =
```

Comment:
What's `DummyImpicit`?

Reply:
Because `PosInt` and `Quantity[PosInt, Picometer]` both erase to `Int` in bytecode the two `apply` methods have the same signature and conflict. The `DummyImplicit` distinguishes them.

Probably we can use `@targetName` annotation here instead.

Reply:
Eh, I forgot we already published 0.51 so now we have compatibility constraints :) a `DummyImplicit` is harmless.